### PR TITLE
Test execution manager

### PR DESCRIPTION
**Pull request by moody6520**
integrate

### CODE REVIEW

File: felucca/backend/execution_manager.py [hunk 1]
```diff
@@ -1,2 +1,209 @@
...
+        """return the command_line_input correspoding to the given task_id
+    
+        Args:
+            task_id (Task): The id of the task
+        
+        Returns:
+            command_line_input (str): The command_line used inside the container to run the executable file with phraos
+        """
+        
+        task = self.map_id2Task_container[task_id][0]
```
Comment (xkbt777) [inline felucca/backend/execution_manager.py:174]: I think the map name is changed.

Comment (jackzihaozhou) [reply felucca/backend/execution_manager.py:174]: solved, thx!

File: felucca/backend/execution_manager.py [hunk 2]
```diff
@@ -1,2 +1,209 @@
...
+from job_manager import JobManager
+
+CONTAINER_PORT = '5000'
+
+@Singleton
 class ExecutionManager(object):
-    pass
\ No newline at end of file
+    
+    """Resource Manager is responsible for:
```
Comment (xkbt777) [inline felucca/backend/execution_manager.py:19]: Wrong comment

**Comment by moody6520** [comment]
Test execution manager successfully !!!!

**Comment by moody6520** [comment]
integrate with other pr
https://github.com/MITS-Felucca/Felucca/pull/120

**Pull request closed without merging**